src/ProjectTemplates/Web.ItemTemplates/content/MvcController/.template.config/template.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "Web", "ASP.NET" ],
+  "name": "MVC Controller",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A MVC Controller with or without read/write actions",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.MvcController",
+  "precedence": "9800",
+  "identity": "Microsoft.AspNetCore.Mvc.MvcController.8.0",
+  "shortName": "mvcController",
+  "sourceName": "HomeController",
+  "primaryOutputs": [
+    {
+      "path": "HomeController.cs"
+    }
+  ],
+  "defaultName": "HomeController",
+  "symbols": {
+    "namespace": {
+      "description": "namespace for the generated code",
+      "replaces": "MyApp.Namespace",
+      "type": "parameter"
+    },
+    "actions": {
+      "description": "create controller with read/write actions",
+      "displayName": "Add ReadWrite Actions",
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "false"
+    },
+    "HostIdentifier": {
+      "type": "bind",
+      "binding": "HostIdentifier"
+    },
+    "NameIsController": {
```

Comment:
I don't see this getting used in the HomeController.cs file.

Reply:
missed it, using it now.